File: Boca/Q11/src/Produto.java
public class Produto {
    private String nome;
    private double preco;

    public Produto (String nomeProduto, double precoProduto) {
        nome = nomeProduto;
        preco = precoProduto;
    }

    public String getNome() {
        return nome;
    }

    public double getPreco() {
        return preco;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public void setPreco(double preco) {
        this.preco = preco;
    }
  
}

File: Boca/CPP/06/Produto.cpp
#include "Produto.hpp"

Produto::Produto(const string& nome, double preco) : nome(nome), preco(preco) {}


void Produto::print(ostream &out) const {
    out << nome << " (R$ " << preco << ")";
}

ostream& operator<< (ostream &out, const Produto &produto) {
    produto.print(out);
    return out;
}

Produto::~Produto() {}
